proto/viam/app/v1/billing.proto

```diff
@@ -71,6 +71,7 @@ message GetCurrentMonthUsageResponse {
   double discount_amount = 8;
   double total_usage_with_discount = 9;
   double total_usage_without_discount = 10;
+  double per_machine_usage_cost = 11;
```

Comment:
I wonder if we should somehow indicate whether or not this field is in use for the current customer. There's a risk of ambiguity if someone has a 0 value for this (did not use their machines) versus being on a different model entirely.

So the question is should this billing.proto communicate what fields are in use given the current tier.

Reply:
yeah agreed it could be ambiguous, but also true of any of the other usage counts in this API now. users can see the breakdown of costs in their invoice (along with the cost per unit). we could also add an a separate endpoint at some point that displays the billing tier information?